docs/index.html

```diff
@@ -72,6 +73,12 @@
   <!-- Rust -->
   <script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-php.min.js"></script>
   <!-- PHP -->
+  <script src="//unpkg.com/prismjs/components/prism-c.min.js"></script>
+  <!-- C -->
+  <script src="//unpkg.com/prismjs/components/prism-cpp.min.js"></script>
+  <!-- C++ -->
```

Comment:
seems like we're switching cdns on these two source urls for no reason, might be best to stick to jsdelivr for all urls or switch to unpkg for all of them
(unless this is done because jsdelivr doesn't have those files)
```suggestion
  <script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-c.min.js"></script>
  <!-- C -->
  <script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-cpp.min.js"></script>
  <!-- C++ -->
```